cli/src/trusted_operation.rs

```diff
@@ -48,20 +48,31 @@ pub(crate) fn perform_trusted_operation(
 	match top {
 		TrustedOperation::indirect_call(_) => send_request(cli, trusted_args, top),
 		TrustedOperation::direct_call(_) => send_direct_request(cli, trusted_args, top),
-		TrustedOperation::get(getter) => get_state(cli, trusted_args, getter),
+		TrustedOperation::get(getter) => get_state_with_new_worker_api(cli, trusted_args, getter),
 	}
 }
 
-fn get_state(cli: &Cli, trusted_args: &TrustedArgs, getter: &Getter) -> Option<Vec<u8>> {
+fn get_state_with_new_worker_api(
```

Comment:
I don't get the name of this function. Why is it new? It's still the `direct_api` ?

Reply:
This function creates a new instance of worker api (as opposed to the other function that takes an existing worker api instance). That's the explanation for the `_with_new_worker_api`. But since that wasn't obvious to you, I should probably find a better name 🙈